src/coaster.cpp

```diff
@@ -23,6 +23,8 @@ CoasterPlatform _coaster_platforms[CPT_COUNT]; ///< Sprites for the coaster plat
 static CarType _car_types[16]; ///< Car types available to the program (arbitrary limit).
 static uint _used_types = 0;   ///< First free car type in #_car_types.
 
+static const uint16 ENTRANCE_OR_EXIT = INT16_MAX;  ///< Indicates that a voxel belongs to an entrance or exit.
```

Comment:
I wonder if it is useful to distinguish between both types by a different value.

Reply:
No point IMHO. This special value is used only to recognize that a certain voxel is not occupied by any track piece while drawing the coaster. In all other places we compare whether one of the ride's entrances/exits is located at the coordinate of interest.